test/unittests/main_UnitTest.cpp

```diff
@@ -110,9 +129,26 @@ int main(int argC, char** argV)
   }
   else
     paParse(paArgs, 1, argV, 1, false);
+#else
+  // FIXME PR: With this change the following works:
+  //
+  // BUILD_UNITTEST/test/unittests/unitTest -t 0-255 -dbhost qa-bigdata-sth-01
+  //
+  // However, if we introduce gtest arguments, it breaks, e.g:
+  //
+  // BUILD_UNITTEST/test/unittests/unitTest -t 0-255 -dbhost localhost --gtest_filter=*.*
+  // ...
+  // option '--' is a boolean option - cannot have a value (gtest_filter=*.*)
+  //
+  // I don't know how to fix this... paArgs wisdom is required :)
+
+  paParse(paArgs, argC, (char**) argV, 1, false);
```

Comment:
Just be sure, with the fix implemented in https://github.com/telefonicaid/fiware-orion/commit/e5391f7e613ffe653e7ddb7e69b4c88cad62e4c7, the `--gtest*` arguments... are accepted and nullified (i.e. they haven't any effect)? Or are they accepted and used "internally" by the unit test library in the way they are expected (e.g. `--gtest_filter` to filter test to execute, etc.)?

I understand is the second one, but looking to the commit is not so clear to me... It would be great to unserstand how to "configure" `paArgs[]` if I need this in some other place.